slither/detectors/abstract_detector.py

```diff
@@ -61,6 +61,11 @@ class AbstractDetector(metaclass=abc.ABCMeta):
 
     STANDARD_JSON = True
 
+    # list of vulnerable solc versions as strings (e.g. ["0.4.25", "0.5.0"])
+    # if this list is not empty then the detector will not run unless the solc version is on the list
+    # an empty list means that the detector will run on any solc version
+    VULNERABLE_SOLC_VERSIONS = []
```

Comment:
Maybe we could add the type here: `List[str]` to avoid confusion

Reply:
good call, added.